SR2020/2020-03-05-coronavirus.md

```diff
@@ -0,0 +1,21 @@
+---
+to: sr2020-teams
+subject: Update on Coronavirus and SR
+---
+
+# Update on SR and the COVID-19 Coronavirus
+
+Hello everyone, 
+
+TL;DR: We're not intending to cancel the event, but will be monitoring the situation closely and will be taking action both during and before the event to minimize risk.
+
+I'm sure you've heard about the threat caused by the COVID-19 coronavirus. and you've likely heard of many large events being cancelled due to concerns of spreading of the virus.
+
+At the current time we're *not* intending to cancel Student Robotics 2020 competition. However, we're keeping a keen eye on the situation, and will be following the advice of the [UK Department of Health and Social Care][uk-coronavirus-page]. We are also working on a back-up plan should the event be unable to run. Considering this is an event for minors, we will be taking decisions with an abundance of caution.
+
+With an abundance of caution, we've decided to take action during the competition to minimise risk. We will frequently disinfect any touch points (i.e. door handles & surfaces) during the competition, and will try to source hand sanitizer for use during the event, (however there is currently a global shortage). We'll also be briefing all volunteers and attendees about best hygene practises. We'd also like to request every attendee who has recently visited or will arrive from another country to follow the 'returning travellers' section of the [UK Government guidance][returning-travellers].
+
+If you or your school has any requirements or questions about our handling of the event, don't hesitate to contact us at teams@studentrobotics.org.
```

Comment:
I wouldn't invite them to add requirements, as that'll encourage them to think of some. Encourage questions, and if their school does happen to have additional requirements they will ask about them.

```suggestion
If you or your school has any questions about our handling of the event, please don't hesitate to contact us at teams@studentrobotics.org.
```